ONSdigital/rm-notify-gateway
# Add basic HTTP auth to headers for requests to Comms Template

# Motivation and Context
We have added basic auth to the Comms Template service, but the Notify Gateway service was not sending the username and password in the headers, causing HTTP 401 errors (unauthorised).

# What has changed
Updated Notify Gateway to use the correct version of the framework utility class project. Updated the code to grab an HttpEntity with the headers containing the username & password.

# How to test?
Force an action to trigger which will cause emails to be sent, and check the logs to ensure that the template has been retrieved from Comms Template without any problems.

# Links
Trello: https://trello.com/c/94Cs866R/550-bug-notify-gateway-does-not-use-http-auth-in-headers-for-requests-to-comms-template
